src/Projection/PdoEventStoreReadModelProjector.php

```diff
@@ -595,8 +596,11 @@ private function createProjection(): void
 EOT;
 
         $statement = $this->connection->prepare($sql);
-        // we ignore any occuring error here (duplicate projection)
-        $statement->execute([$this->name, $this->status->getValue()]);
+        try {
+            $statement->execute([$this->name, $this->status->getValue()]);
+        } catch (PDOException $e) {
```

Comment:
I actually wanted to solve the same issue. The problem with checking error codes is that you get different ones depending on the driver. 
```php
[
   "pgsql" => "23505",
   "mysql" => "1062"
]
```

I could make a new PR with this in mind (I don't think I can add all the error codes for all the drivers, but new PR could solve this). @prolic Would you accept this solution ?